lib/utils/GroupHelper.js

```diff
@@ -100,6 +98,10 @@ class GroupHelper {
         }
         return configPathExists ? predefinedConfigPath : configPath;
     }
+
+    run() {
```

Comment:
A class that extends `GroupHelper` *must* implement the `run` function

Reply:
This check should be in constructor of the class.

Reply:
No, this implementation is "a la TypeScript". `GroupHelper` is like our abstract class. If we create a new group and we forget to implement `run` an error is thrown.

Reply:
Oh, I see now `run()` is like a pure virtual function (right?). Lets keep this then.

Reply:
Yes exactly!! :)